contracts/contracts/core/ProviderRegistry.sol

```diff
@@ -126,7 +128,7 @@ contract ProviderRegistry is
             bidderSlashedAmount[bidder] += residualAmt;
         }
 
-        emit FundsSlashed(provider, residualAmt + penaltyFee);
+        emit FundsSlashed(provider, residualAmt + penaltyFee, commitmentDigest);
```

Comment:
So I thought what we discussed is emitting the new event and not changing the old one. If we change the old one I am not sure how backward compatibility works. For eg, if they listen to old contract events with the new ABI what would happen? Would it be able to fill only the relevant items?